src/compiler/checker.ts

```diff
@@ -28997,7 +28999,7 @@ namespace ts {
                     }
                     return isErrorType(apparentType) ? errorType : apparentType;;
                 }
-                prop = getPropertyOfType(apparentType, right.escapedText);
+                prop = getPropertyOfType(apparentType, right.escapedText, /*skipObjectFunctionPropertyAugment*/ false, /*includeTypeOnlyMembers*/ node.kind === SyntaxKind.QualifiedName);

```

Comment:
Nit: is there a reason why here you used `/*skipObjectFunctionPropertyAugment*/ false` but above you use `undefined`?

Reply:
Just to see if you were paying attention